core/src/repair_service.rs

```diff
@@ -29,22 +29,39 @@ use std::{
 pub type DuplicateSlotsResetSender = CrossbeamSender<Slot>;
 pub type DuplicateSlotsResetReceiver = CrossbeamReceiver<Slot>;
 
-#[derive(Default)]
+#[derive(Default, Debug)]
+pub struct SlotRepairs {
+    highest_shred_index: u64,
+    // map from pubkey to total number of requests
+    pubkey_repairs: HashMap<Pubkey, u64>,
```

Comment:
needs to get cleared somewhere?

Reply:
All of this gets deleted when we reset here: https://github.com/solana-labs/solana/pull/10717/files#diff-f03a09b141837083bc32c6373631d89fR229 right?